### DISCUSSION TITLE
Goety

### DISCUSSION THREAD

**Discussion by ThisJustInLocal** [DynamicTreesTeam/DynamicTrees#951] [Compatibility Addon Suggestions]
- Mod Link: [https://www.curseforge.com/minecraft/mc-mods/goety](url)

- Biomes: None.

- Trees: Three, which are: Windswept, Rotten, and Haunted.

- Preferred Version: 1.20.1, although mod spans back to 1.16.5.
![2024-10-09_14 53 43](https://github.com/user-attachments/assets/00e8a32c-5edf-4b36-9913-2f54d80aa182)

**Comment by Ruk11571** [comment]
The link is broken
